google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java

```diff
@@ -389,7 +389,8 @@ public Builder toBuilder() {
     private final UnaryCallSettings.Builder<Query, Row> readRowSettings;
     private final UnaryCallSettings.Builder<String, List<KeyOffset>> sampleRowKeysSettings;
     private final UnaryCallSettings.Builder<RowMutation, Void> mutateRowSettings;
-    private final BatchingCallSettings.Builder<RowMutation, Void> bulkMutateRowsSettings;
+    private final BatchingCallSettings.Builder<RowMutationEntry, Void, RowMutation, Void>
```

Comment:
Why is the request type RowMutation? Shouldn't it be BulkMutation?

Reply:
Actually, I mistakenly assumed that we should not deprecate and change the Request wrapper RowMutation to BulkMutation in one go.

But I have updated it after you pointed it out.